src/security/builtin_plugins/cryptographic/src/crypto_transform.c

```diff
@@ -331,7 +331,7 @@ static bool read_submsg_header (tainted_input_buffer_t *input, uint8_t smid, dds
   // silly C can't deal with assignment to *submsg_view in any way because of endp
   // memcpy to the rescue!
   // coverity[store_writes_const_field]
-  memcpy (submsg_view, &(tainted_input_buffer_t){ .ptr = input->ptr, .endp = input->ptr + hdr->octetsToNextHeader }, sizeof (*submsg_view));
+  memcpy (submsg_view, (&(tainted_input_buffer_t){ .ptr = input->ptr, .endp = input->ptr + hdr->octetsToNextHeader }), sizeof (*submsg_view));
```

Comment:
Interesting ... I guess `memcpy` was a macro ...

I think it needs a comment that the parentheses are not superfluous, otherwise someone (me, perhaps) might just remove them again.

Reply:
Yes, see: https://github.com/Mbed-TLS/mbedtls/issues/8994